libcxx/test/std/atomics/atomics.lockfree/is_always_lock_free.pass.cpp

```diff
@@ -5,8 +5,9 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-//
+
 // UNSUPPORTED: c++03, c++11, c++14
+// XFAIL: LIBCXX-PICOLIBC-FIXME
```

Comment:
@DavidSpickett

Reply:
I will merge this because we want to cherry-pick onto LLVM 19 and it's better to have this test for most platforms than for none of them, but I would really appreciate if we could address this in a follow-up.